package.json

```diff
@@ -31,7 +34,15 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2",
     "jest": "^24.9.0",
+    "nodemon": "^1.19.4",
     "stylelint": "^11.1.1",
-    "stylelint-config-recommended": "^3.0.0"
+    "stylelint-config-recommended": "^3.0.0",
+    "supertest": "^4.0.2"
+  },
+  "jest": {
```

Comment:
This seems unrelated to your changes.  We have an issue on #120, can you comment there, and they can add this?

Reply:
yup! I'll remove it from here and comment in #120